src/backend/shortcuts/nonesteamgame/steamhelper.ts

```diff
@@ -52,15 +53,35 @@ async function prepareImagesForSteam(props: {
 
   const errors: string[] = []
   const images = new Map<string, string>([
-    [coverArt, props.gameInfo.art_square],
-    [headerArt, props.gameInfo.art_cover],
-    [backGroundArt, props.bkgDataUrl],
+    [
+      coverArt,
+      props.steamID
+        ? `https://cdn.cloudflare.steamstatic.com/steam/apps/${props.steamID}/library_600x900_2x.jpg`
```

Comment:
maybe instead of repeating the whole URL, add it to a variable since it is the same, like baseURL or something and add the rest.